<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExpenseWindow</class>
 <widget class="QWidget" name="ExpenseWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>290</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Date of Expense</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextDay">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>370</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Day</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="currentMonth">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>710</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="previousDay">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>400</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous Day</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>670</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Local Time</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dailyExpenses">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>330</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>741</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>This window will be used for at once committing all expenses for a whole month</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>230</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Input your daily expenses below</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="startOfCurrentMonth">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>330</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>290</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Daily Expense</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>690</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
